### ISSUE TITLE
New Camo Not loading causing invisible vehicles

### ISSUE THREAD

**Issue by Classified787** [A3Wasteland/ArmA3_Wasteland.Altis#202]
I am having a problem with the new camo in the stores not loading on my server and causing the purchased vehicles being invisible.
On the custom camo I made for my server after a restart the camo would disappear but the vehicle would default back to the vehicles default camo. 
Is any one else having this issue on their servers?

**Comment by AgentRev** [comment]
I think I have found the source of the problem; it seems to be a bug in the netcode of the setObjectTextureGlobal command. When applying a mission-specific texture to an object, the absolute Arma path is added in front of the local mission part, and broadcast directly as-is to all clients. For the server, this might look like "mpmissions\A3Wasteland.Altis\client\images\vehicleTextures\nato.jpg". However, sometimes clients don't use the same path to refer to the location of the mission; the "cannot load texture" error happens when the client uses the generic path "mpmissions\__CUR_MP.Altis\", which happens most often when server runs off a direct mission folder instead of a PBO, but it does sometimes occur even with a proper PBO.

tl;dr : shitty netcode courtesy of BIS

**Comment by Classifed787** [comment]
That is what I am doing, running the server from a direct folder instead of PBO. So should I pack it back into a PBO and run it that way. It is just easier to mess with when I have to do a small edit when it is in a folder format on the server versus the PBO.

**Comment by AgentRev** [comment]
Yeah, but it has the disadvantage of causing players to redownload the mission everytime they join, as opposed to only since last edit.

**Comment by osuapoc** [comment]
My experience is that they only download the PBO when it is changed, same as when it is in folder form.  This may not be the case if they join a different server between your server joins. 

-----Original Message-----

**Comment by LouDnl** [comment]
This is my experience aswell. I run from a folder and not a PBO as it gives more freedom when editing small parts. Players only have to redownload if anything has changed in the folder.

**Comment by AgentRev** [comment]
Yeah, I guess you only have to redownload if you switch back and forth from another server which also runs from a folder.

**Comment by AgentRev** [comment]
Created a bug report: http://feedback.arma3.com/view.php?id=22310

**Comment by LouDnl** [comment]
So for now it's better to disable the textures?

**Comment by AgentRev** [comment]
That or start using a PBO, you decide.

**Comment by Classifed787** [comment]
Have you seen the same problem with the new PAA files for the new textures in V1.1?
So far they have not done it for me. But then it just came out two  days ago.
